### PR TITLE
01 maven

### PR DESCRIPTION
**Pull request by VarvaraKrestyanova**
multi-module maven project structure - 01_maven, 02_git

### CODE REVIEW

File: online-store/.idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (artemmoroz) [inline online-store/.idea/.gitignore:1]: please use gitignore from here https://www.toptal.com/developers/gitignore  or create proper one for java and ide yourself to exculed user specific files.

Comment (artemmoroz) [reply online-store/.idea/.gitignore:1]: to apply this changes google apply gitignore on existing repo

File: online-store/parent/pom.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
```
Comment (artemmoroz) [inline online-store/parent/pom.xml:5]: online store is alredy a parent, no need to create one more parent module.

File: online-store/pom.xml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      <module>consoleApp</module>
+    </modules>
+
+    <name>online-store</name>
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
```
Comment (artemmoroz) [inline online-store/pom.xml:24]: why do you use 1.7 java here, and 1.8 in modules?

File: online-store/pom.xml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    <maven.compiler.target>1.7</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+    <dependency>
```
Comment (artemmoroz) [inline online-store/pom.xml:29]: you do not need junit for now.

**Review by artemmoroz — CHANGES_REQUESTED** [review]

File: online-store/.idea/.gitignore (hunk 1 above)
Comment (VarvaraKrestyanova) [reply online-store/.idea/.gitignore:1]: changes are in a new commit

**Pull request merged**